packages/thefuck.rb

```diff
@@ -0,0 +1,14 @@
+require 'package'
+
+class thefuck < Package
+  description 'Magnificent app which corrects your previous console command.'
+  homepage 'https://github.com/nvbn/thefuck'
+  version '3.26'
+
+  depends_on 'python3'
+
+  def self.install
+    system "python3 -m pip install thefuck --root=#{CREW_DEST_DIR} --prefix=#{CREW_PREFIX}"
```

Comment:
Actually, I think the `=`s are unneccessary.  Can you please test?

Reply:
what are you referring to by "the `=`s"?

Reply:
I think this would be a simpler install line:
```
system "pip3 install thefuck --root #{CREW_DEST_DIR} --prefix #{CREW_PREFIX}"
```

Reply:
Also, do we need setuptools?  `pip3 install setuptools`  Can you please try `pip3 uninstall --yes setuptools` and see if it installs correctly?

Reply:
setuptools is not in the dependency list for thefuck. It installed without it on my chromebook. I forgot `pip3` was a thing. I have been doing it the long way forever. I will fix that now.

Reply:
oh. I have the package cached. setuptools is actually needed.